src/BenchmarkGrainInterfaces/MapReduce/DataflowGrainsInterfaces.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans;
+
+namespace GrainInterfaces
+{
+    public interface IDataflowGrain : IGrainWithGuidKey
```

Comment:
It is interesting that this is imitating the DataFlow library. Nevertheless, is this valuable for the kind work you are doing or is it mainly a cool sample?
I'm not pushing back, just want to understand why we are adding it.